libguard/guard_interface.cpp

```diff
@@ -159,6 +159,7 @@ GuardRecord create(const EntityPath& entityPath, uint32_t eId, uint8_t eType)
                 offset = lastPos * sizeOfGuard;
                 existGuard.errType = eType;
                 existGuard.recordId = htobe32(lastPos + 1);
+                existGuard.elogId = htobe32(eId);
                 file.write(offset + headerSize, &existGuard, sizeOfGuard);
             }
             else if (existGuard.recordId == GUARD_RESOLVED)
```

Comment:
What happens in the below case?

Host created the Reconfig type guard record and BMC tried to clear in the next IPL if the cold boot and normal ipl.

Now, some applications trying to create the guard record on the same resource, what will happen, do we override the existing reconfig record, or will create a new one? I think it should be a new one since it is resolved.